src/ui.cxx

```diff
@@ -1145,6 +1145,8 @@ void GetParamFile(Options &opt)
                         opt.impiusemesh = (atoi(vbuff)>0);
                     else if (strcmp(tbuff, "MPI_zcurve_mesh_decomposition_min_num_cells_per_dim")==0)
                         opt.minnumcellperdim = atoi(vbuff);
+                    else if (strcmp(tbuff, "MPI_zcurve_mesh_decomposition_num_cells_per_dim")==0)
```

Comment:
if this is a new option it would need documentation (don't remember where that lives). Also, is it being already dumped when writing a configuration file from the Options?

Reply:
This is the portion that I need to check, hence it still being in draft. The documentation would also need a change.